ui/components/multichain/token-list/token-list.scss

```diff
@@ -0,0 +1,11 @@
+.token-list {
+  &__container {
+    cursor: pointer;
+
+    &--cell {
+      &--text-container {
+        width: 100%;
```

Comment:
I tried to use as="button" here and with that we can get rid of the cursor. But in that case, we will have to fix the css because of the button Element.